src/js/mobile-dropdown.js

```diff
@@ -53,7 +53,7 @@ export function createMobileDropdown(products, isSliding) {
         <button href="#about" class="p-navigation__link global-nav__header-link-anchor" aria-controls="about">
           About
         </button>
-        <ul id="about" class="p-navigation__dropdown ${
+        <ul id="about" class="p-navigation__dropdown" ${
           isSliding ? 'aria-hidden="true"' : ''
         } u-no-margin--bottom">
```

Comment:
Is this a typo? I guess `u-no-margin--bottom` should sit inside the `class` attribute
```suggestion
        <ul id="about" class="p-navigation__dropdown  u-no-margin--bottom" ${
          isSliding ? 'aria-hidden="true"' : ''
        }>
```

Reply:
Good catch. Thanks!